### ISSUE TITLE
Add focus trapping to v1 side-nav

### ISSUE THREAD

**Issue by sgomes** [google/material-design-lite#4951]
The v1 side-nav is missing focus trapping, so it's currently possible to tab out of the drawer and into the background content, while it's open. We should implement focus trapping to prevent that from happening.

**Comment by sgomes** [comment]
Openly looking for PRs on this one :)

**Comment by Aribaskar-jb** [comment]
hi @sgomes i am new to open Source you can help me to get a PR on this Issue

**Comment by Aribaskar-jb** [comment]
and you can briefly explain me the issue..
